apps/chat/src/components/Chat/Chat.tsx

```diff
@@ -146,6 +150,9 @@ const ChatView = memo(() => {
   const [isLastMessageError, setIsLastMessageError] = useState(false);
   const [prevSelectedIds, setPrevSelectedIds] = useState<string[]>([]);
   const [inputHeight, setInputHeight] = useState<number>(142);
+  const [isWideLayout, setIsWideLayout] = useState(
+    !isWideScreen() && !mergedMessages.length && !isCompareMode,
```

Comment:
`isWideLayout` should be for `isWideScreen()`. Right?
```suggestion
    isWideScreen() && !mergedMessages.length && !isCompareMode,
```